database/factories/TitleFactory.php

```diff
@@ -22,7 +22,9 @@ class TitleFactory extends Factory
     public function definition()
     {
         return [
-            'title' => $this->faker->streetName
+            'title' => $this->faker->streetName,
+            'genre_id' => \App\Models\Genre::inRandomOrder()->value('id'), //PRIMARY GENRE
+            'user_id' => \App\Models\User::inRandomOrder()->value('id'), //is this needed?
```

Comment:
What usage could it have?

Reply:
I believe the 'title' column is the name of the movie or show! :) so it needs to remain i think?

Reply:
meant more the "user_id" the //is this needed?

Reply:
oh yeah, i mean maybe it's fine to keep! if we have multiple admins, could be useful to have a record about which admin made what.